content/docs/troubleshooting/_index.md

```diff
@@ -280,3 +280,64 @@ field once the Ingress resource is ready to route traffic.
 For the Traefik controller, verify that the `kubernetes.ingressEndpoint` config
 is [set properly](https://docs.traefik.io/providers/kubernetes-ingress/). This option was
 introduced in Traefik 1.7.0.
+
+## Synchronous call made to "X" with an unregistered provider {#synchronous-call}
+
+The warning occurs when invoking a resource function synchronously while also using
```

Comment:
Let's standardize on "resource function" or "data source" (and not "data-source"). I don't have a strong opinion on which we choose.

Reply:
Done.